searx/settings.yml

```diff
@@ -326,6 +326,11 @@ categories_as_tabs:
   social media:
 
 engines:
+  - name: baidu
+    engine: baidu
+    shortcut: baidu
+    disabled: false
```

Comment:
Maybe this should be disabled by default, or set to `inactive` because it's probably going to be ratelimited soon otherwise for most public instances?